news/2017-10-11-qat-gazette-october-2017.md

```diff
@@ -0,0 +1,68 @@
+
+Should they demonstrate an acceptable level of activity and quality in this time, they will become full members of the Beatmap Nominators who can both nominate and qualify beatmaps.
+
+### The Qualification Cap
+
+With the removal of the Beatmap Nominator Tier system, the amount of Qualified beatmaps literally doubled in the space of just one month, if not more! This ended in a Qualified stage for beatmaps lasting for up to 12 days or even more. Since this was awfully slow and unproductive, **we raised the amount of beatmaps that can pass from Qualified to Ranked per day to 8.** This will shorten the time that a beatmap set will remain in qualified hopefully back down towards the target of 7 days.
+
+## Within the Quality Assurance Team
+
+There have also been changes within the QAT too. Our team member [Cryptic](https://osu.ppy.sh/users/3728904) decided to step down from the QAT on his own after a short tenure to focus on IRL duties. You will be missed Cryptic, thanks for your contributions!
+
+It’s not all sad news however, as we also have **two new additions too!** Please give a warm welcome to the Quality Assurance Team for [Naxess](https://osu.ppy.sh/users/8129817) and [Raiden](https://osu.ppy.sh/users/2239480)! Raiden has joined to help manage the osu!taiko Beatmap Nominators who have grown a lot with the last phase of applications. Together with Mao, Naxess will assist with development of the Beatmap Nominator subdivisions. But you will hear more about this in future!
+
+## Mapping with Rewards is back!
+
+Some of you might remember the old Mapping with Rewards event, but [Okorin](https://osu.ppy.sh/users/1623405) has recently taken the lead and revived it. You can now choose from three awesome songs to create a beatmap set in any mode of your choosing for this contest. All you have to do is map them and submit - that’s it! Your set can be even ranked already when entering. After the 3 month deadline has passed, a team of dedicated judges will review the entries and decide which is the best.
+
+The creator of the winning beatmap set for each game mode will receive several rewards, such as a 6 months of osu! supporter, a profile badge and more! If  you haven’t read it yet make sure to head over to the [Announcement Page](https://osu.ppy.sh/home/news/2017-09-23-mapping-with-rewards-returns) and read more about it!
+
+If you want even more mapping events, [Irreversible](https://osu.ppy.sh/users/1287964) has something special planned for the coming month, so stay tuned!
+
+## Summary
+
+That’s it! I hope you enjoyed reading this month’s issue of the QAT Gazette as much i had fun writing it! (No, I really had fun writing it!) As always, if you have any questions about mapping, modding or other related topics you wish to ask us about, make sure to leave it below!
```

Comment:
as much i had fun writing it → as much as I had fun writing it (as + capitalisation of I)

Reply:
see above